resources/patches/ruby_241/aix_ruby_2.1_fix_proctitle.patch

```diff
@@ -0,0 +1,30 @@
+--- ruby-1.9.3-p547/missing/setproctitle.c	2013-02-05 21:13:40.000000000 -0800
++++ ruby-1.9.3-p547-patched/missing/setproctitle.c	2014-10-08 09:58:12.000000000 -0700
```

Comment:
I don't think we need this patch anymore. Ruby has these changes in mainline since 2.2.0: https://github.com/ruby/ruby/blob/trunk/missing/setproctitle.c#L124-L126. Not sure how long those have